### PR TITLE
feat: lesson 10 homework

### PR DESCRIPTION
**Pull request by trongtai37**

### CODE REVIEW

File: homework/tai_nguyen/lesson10/JS_questions.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+funcOne();
+funcTwo();
+```
+
+- A: Promise 1! Last line 1! Promise 2! Last line 2! Timeout 1! Timeout 2!
+- B: Last line 1! Timeout 1! Promise 1! Last line 2! Promise2! Timeout 2!
+- C: Last line 1! Promise 2! Last line 2! Promise 1! Timeout 1! Timeout 2!
+- D: Timeout 1! Promise 1! Last line 1! Promise 2! Timeout 2! Last line 2!
+
+> Answer C
```
Comment (thaycacac) [inline homework/tai_nguyen/lesson10/JS_questions.md:202]: Explain this question, pls @trongtai37

**Comment by thaycacac** [comment]
No answer yet for Q3. other question LGTM.

File: homework/tai_nguyen/lesson10/JS_questions.md (hunk 1 above)
Comment (trongtai37) [reply homework/tai_nguyen/lesson10/JS_questions.md:202]: Sorry @thaycacac, my last week was busy and I have not has time to explain all the questions yet.
This PR is temporarily draft and I will inform you when finishing these JS questions ASAP.
Thank you for your early review.

Comment (thaycacac) [reply homework/tai_nguyen/lesson10/JS_questions.md:202]: thanks @trongtai37

Comment (trongtai37) [reply homework/tai_nguyen/lesson10/JS_questions.md:202]: Hi @thaycacac, I have finished the all JS questions.
Please review it for me.
:D

**Comment by thaycacac** [comment]
@quanghoang33 FE part LGTM

**Review by thaycacac — APPROVED** [review]

**Pull request merged**